model/src/test/kotlin/AdvisorResultContainerTest.kt

```diff
@@ -90,6 +92,22 @@ class AdvisorResultContainerTest : WordSpec() {
 
                 deserializedAdvisorResults shouldBe advisorResults
             }
+
+            "be deserialized with vulnerabilities in the initial format" {
+                val resultsFile = File("src/test/assets/advisor-result-initial.yml")
+
+                val result = resultsFile.readValue<OrtResult>()
+
+                result.advisor.shouldNotBeNull()
+            }
+
+            "be deserialized with vulnerabilities in format with references" {
+                val resultsFile = File("src/test/assets/advisor-result-vulnerability-refs.yml")
+
+                val result = resultsFile.readValue<OrtResult>()
+
+                result.advisor.shouldNotBeNull()
```

Comment:
Please prefer to use the infix notation.

Reply:
See comment above.